docs/guides/chip_tool_guide.md

```diff
@@ -382,34 +395,42 @@ $ ./chip-tool basic
 This section contains a general list of various CHIP Tool commands and options,
 not limited to commissioning procedure and cluster interaction.
 
-### Interactive mode versus single command mode
+### Interactive mode versus single-command mode
 
-By default, chip-tool runs in single command mode where if any single command
-does not complete within a certain timeout period, chip-tool will exit with a
-timeout error.
+The CHIP Tool can run in one of the following modes:
 
-Example of error:
+-   Single-command mode (default) - In this mode, the CHIP Tool will exit with a
+    timeout error if any single command does not complete within a certain
+    timeout period.
 
-```
-[1650992689511] [32397:1415601] CHIP: [TOO] Run command failure: ../../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout
-```
+    The timeout error will look similar to the following one:
+
+    ```
+    [1650992689511] [32397:1415601] CHIP: [TOO] Run command failure: ../../../examples/chip-tool/commands/common/CHIPCommand.cpp:392: CHIP Error 0x00000032: Timeout
+    ```
+
+-   Interactive mode - In this mode, there is no timeout and multiple commands
+    can be issued.
```

Comment:
This is not accurate.  In interactive mode there is still a per-command timeout, and a command that does not complete within the timeout will be terminated with an error.  But that will not terminate chip-tool itself and will not terminate async things that previous commands have started.

Reply:
Rephrased.